org.robiot.xclicker.yml

```diff
@@ -0,0 +1,78 @@
+app-id: org.robiot.xclicker
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: xclicker
+desktop-file-name-suffix: " (Flatpak)"
```

Comment:
```suggestion
```

Reply:
Well yeah, but you can also install it without flatpak and that would result in you having 2 identical apps in your app launcher. But sure, it can be removed.